samples/sensor/bme280/sample.yaml

```diff
@@ -24,3 +24,26 @@ tests:
       regex:
         - "temp: (.*); press: (.*); humidity: (.*)"
       fixture: fixture_spi_bme280
+  sample.sensor.bme280.rpi_pico:
+    harness: console
+    tags: sensors
+    platform_allow: rpi_pico
+    integration_platforms:
+      - rpi_pico
+    harness_config:
+      type: one_line
+      regex:
+        - "temp: (.*); press: (.*); humidity: (.*)"
+      fixture: fixture_spi_bme280
```

Comment:
This seems exactly the same as `sample.sensor.bme280` with the exception of adding the rpi_pico to the platforms. You can just edit that node to look like:

```
platform_allow:
  - adafruit_feather_m0_basic_proto
  - rpi_pico
```

Since integration _platforms is the same as the platform_allow, you can really just get rid of it.